devops/jobs/CloudFlarePurgeCacheOrigin.groovy

```diff
@@ -0,0 +1,95 @@
+/*
+
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    Variables consumed for this job:
+        * CLOUDFLARE_EMAIL: (Required - email address)
+        * CLOUDFLARE_API_KEY:
+        * ORIGIN: (Required - jenkins name of git credentials)
+        * CLOUDFLARE_ZONE_ID: (Required - The CloudFlare Zone ID)
+        * TARGET_PATH: (Required - Authentication key for the account that would make API calls)
+        * CLOUDFLARE_SITE_URL: (Required - Email of the account for making API calls)
```

Comment:
Could we update the docs/code to have matching variable names (e.g. CLOUDFLARE_API_KEY, CLOUDFLARE_EMAIL NOTIFY_ON_FAILURE)?  There seem to also be one or two that may not be needed at all as well.